### PR TITLE
Remove ropsten/rinkeby references

### PR DESCRIPTION
**Pull request by Siegrift**
Closes https://github.com/api3dao/airnode/issues/1280

### CODE REVIEW

File: packages/airnode-examples/src/scripts/choose-integration.ts [hunk 1]
```diff
@@ -79,18 +79,16 @@ const questions: PromptObject[] = [
...
+      switch (values.network as IntegrationInfo['network']) {
         case 'localhost':
           return 'http://127.0.0.1:8545/';
-        case 'rinkeby':
-        case 'ropsten':
         case 'goerli':
         case 'kovan':
           return getExamplePocketNetwork(values.network);
-        case 'polygon-mumbai':
+        case 'polygon-testnet':
```
Comment (Siegrift) [inline packages/airnode-examples/src/scripts/choose-integration.ts:88]: This was a bug before.

**Comment by Siegrift** [comment]
I've tested using the docker container version airnode-examples. Works fine.

**Review by aquarat — APPROVED** [review]
LGTM 🔥 🚒

File: packages/airnode-examples/src/scripts/choose-integration.ts (hunk 1 above)
Comment (dcroote) [reply packages/airnode-examples/src/scripts/choose-integration.ts:87]: Nice 👍

**Pull request merged**